tasks/main.yml

```diff
@@ -3,34 +3,58 @@
   fail: msg="ansistrano_custom_tasks_path is not used anymore. Please, check the documentation at https://github.com/ansistrano/deploy"
   when: ansistrano_custom_tasks_path is defined
 
-- name: ANSISTRANO | Execute "Before Setup" Tasks
-  include: "{{ ansistrano_before_setup_tasks_file | default('empty.yml') }}"
+- include: "{{
+    ansistrano_before_setup_tasks_file
+    | default(ansistrano_display_default_hooks is defined and ansistrano_display_default_hooks)
```

Comment:
`{{ some_var | default('hello') }}` evaluates as expected

`{{ some_var | default(another_var) }}` errors out, does not evaluate to the contents of another_var

which brings us to this use of boolean > ternary